src/angular/controllers/ctrl-widget-catalog.test.js

```diff
@@ -0,0 +1,300 @@
+describe('WidgetCatalogCtrl', () => {
```

Comment:
Since the widget catalog and details pages have been completely rewritten in React, this file is no longer necessary. I'll make a separate commit to remove it.